utils/github-api-helpers.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const fetch = require('node-fetch');
+const parseLinkHeader = require('parse-link-header');
+
+const CONFIG_REGEXP = new RegExp('quickstarts/.+/config.+(yml|yaml)');
```

Comment:
i see `QUICKSTART_CONFIG_REGEXP` is referenced on line 59, should this match?
```suggestion
const QUICKSTART_CONFIG_REGEXP = new RegExp('quickstarts/.+/config.+(yml|yaml)');
```

Reply:
Yes, thanks! c5c2c71c